app/sender.py

```diff
@@ -6,12 +6,14 @@
 import heapq
 from typing import TYPE_CHECKING, Protocol
 
+from app.database import UserProfile
+
 if TYPE_CHECKING:
     from collections.abc import Awaitable, Callable
 
     import discord
 
-    from __main__ import DiscordClient
+    from app.__main__ import DiscordClient
```

Comment:
Don't add these `app.` things that depend on CWD. I tested with `mypy` and all we need to do is delete `__init__` -- I will do that soon.